cmd/dockerfile/digests.go

```diff
@@ -0,0 +1,119 @@
+package dockerfile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+	"sync"
+
+	"github.com/buildsafedev/bsf/cmd/styles"
+	"github.com/google/go-containerregistry/pkg/crane"
+	"github.com/spf13/cobra"
+)
+
+var DGCmd = &cobra.Command{
+	Use:     "digests",
+	Short:   "Replace Dockerfile image tags with immutable digests",
+	Aliases: []string{"dg"},
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			fmt.Println(styles.HintStyle.Render("hint:", "run `bsf dockerfile digests <Dockerfile>` to replace image tags with digests"))
+			os.Exit(1)
+		}
+
+		dockerfile := args[0]
+		file, err := os.ReadFile(dockerfile)
+		if err != nil {
+			fmt.Printf("Error opening Dockerfile: %v\n", err)
```

Comment:
we can use `styles.ErrorStyle.Render` over here and for error messages below in the cmd.

Reply:
should I do this for all errors?